acanban/__main__.py

```diff
@@ -23,6 +24,10 @@
 from . import app
 
 if __name__ == '__main__':
+    parser = ArgumentParser('Additional configuration for the server.')
```

Comment:
The following, or [something that produces `python -m acanban`](https://git.sr.ht/~cnx/offre/tree/a01a3874d74581a085c27492c121c66dd2e5554f/offre/__main__.py#L33).

```suggestion
    parser = ArgumentParser(prog='acanban')
```